app/_src/gateway-operator/reference/version-compatibility.md

```diff
@@ -14,6 +14,7 @@ The following table presents the general compatibility of {{site.kgo_product_nam
 | {{ site.kic_product_name }} 2.12 | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-times"></i> |
 | {{ site.kic_product_name }} 3.0  | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-times"></i> |
 | {{ site.kic_product_name }} 3.1  | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> |
+| {{ site.kic_product_name }} 3.2  | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> | <i class="fa fa-check"></i> |
```

Comment:
This table and the one below it: do they both need a new 1.3.x column?

Reply:
Added the new column for 1.3.x.